orga/app/styles/globals/competences.scss

```
@@ -8,7 +8,7 @@
     position: relative;
     padding-left: 1rem;
     font-family: $open-sans;
-    font-size: 1rem;
+    font-size: 14px;
```

On préfère écrire les font-size en rem
```suggestion
    font-size: 0.875rem;
```
Voilà un outil pour faire le produit en croix :) https://www.jarrige.fr/pixelrem/ (sachant que 1rem = 16px)

---

J'ai fait la modification pour relancer la CI, car on dirait qu'il y a un souci avec le déploiement